devices: unresolved imports `nix::unistd::dup`, `nix::fcntl::fcntl`, `nix::fcntl::FcntlArg`, `nix::fcntl::OFlag`

Sub-Build fails `~/libkrun/src/devices $ cargo build` fails with error:

```Rust
error[E0432]: unresolved import `nix::unistd::dup`
   --> src/devices/src/virtio/console/port_io.rs:9:5
    |
9   | use nix::unistd::dup;
    |     ^^^^^^^^^^^^^^^^ no `dup` in `unistd`
    |

error[E0432]: unresolved imports `nix::fcntl::fcntl`, `nix::fcntl::FcntlArg`, `nix::fcntl::OFlag`
   --> src/devices/src/virtio/vsock/tcp.rs:9:18
    |
9   | use nix::fcntl::{fcntl, FcntlArg, OFlag};
    |                  ^^^^^  ^^^^^^^^  ^^^^^ no `OFlag` in `fcntl`
    |                  |      |
    |                  |      no `FcntlArg` in `fcntl`
    |                  no `fcntl` in `fcntl`
    |
```

--- a/src/devices/Cargo.toml
+++ b/src/devices/Cargo.toml
@@ -24,7 +24,7 @@ crossbeam-channel = ">=0.5.15"
 libc = ">=0.2.39"
 libloading = "0.8"
 log = "0.4.0"
-nix = { version = "0.30.1", features = ["ioctl", "net", "poll", "socket"] }
+nix = { version = "0.30.1", features = ["ioctl", "net", "poll", "socket", "fs"] }
 pw = { package = "pipewire", version = "0.8.0", optional = true }
 rand = "0.9.2"
 thiserror = { version = "2.0", optional = true }

